configs/default/TMTR-TMVeloxF7.config

```diff
@@ -0,0 +1,144 @@
+# Betaflight / STM32F7X2 (S7X2) 4.2.5 Nov 22 2020 / 18:39:53 (afdac08b3) MSP API: 1.43
+board_name TMVeloxF7
```

Comment:
There are places like OSD that cannot display lower case characters. Please use only upper case for board names.

Reply:
OK, I will correct it